instrumentation/src/splunk.c

```diff
@@ -8,14 +8,17 @@
 #include <unistd.h>
 #include <stdio.h>
 
-#define ENV_VAR_LEN 512
+#define MAX_ENV_VAR_LEN 512
```

Comment:
Is this a known limit for `JAVA_TOOL_OPTIONS`?

Reply:
No, it's just an arbitrary value that's roughly an order of magnitude larger than what it will need.

Reply:
SO suggests it's hardcoded to 1024 in the jdk: https://stackoverflow.com/a/41792192